### ISSUE TITLE
info/help icons should have alt-text

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1171]
currently the info/help icons content are only available on hover, which is not accessible to keyboard or screenreader users. The info/help icons are generally also not tab navigable.

**Comment by sudo-may** [comment]
PR: https://github.com/NationalSecurityAgency/skills-service/pull/1341

**Comment by mosefer** [comment]
:+1:
